components/brave_wallet/browser/zcash/zcash_rpc.cc

```diff
@@ -210,6 +242,23 @@ const GURL MakeGetTransactionURL(const GURL& base_url) {
   return base_url.ReplaceComponents(replacements);
 }
 
+const GURL MakeGetCompactBlocksURL(const GURL& base_url) {
+  if (!base_url.is_valid()) {
+    return GURL();
+  }
+  if (!UrlPathEndsWithSlash(base_url)) {
+    return GURL();
+  }
+
+  GURL::Replacements replacements;
+  std::string path =
+      base::StrCat({base_url.path(),
```

Comment:
in principle it's fine for appending to the path, but are we expecting the path from `base_url` to look like here? Are we expecting it to end with a `/`? If so, that doesn't seem ideal to me and it would be better to add `/` at the beginning of what we're appending